node-module-system/calculate.js

```diff
@@ -0,0 +1,14 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+if (process.argv.includes('plus')) {
+  console.log('results:', add(Number(process.argv[2]), Number(process.argv[4])));
```

Comment:
You repeat `process.argv[2]`, `process.argv[4]`, and `process.argv` repeatedly. It would make your code cleaner if you extracted those as variables. Then you wouldn't have to repeat yourself over and over.